g2d-skijagraphics2d/src/main/java/org/jfree/graphics2d/tester/skija/SkijaGraphics2DTestRunner.java

```diff
@@ -2,36 +2,62 @@
 
 import io.github.humbleui.skija.Data;
 import io.github.humbleui.skija.EncodedImageFormat;
+import io.github.humbleui.skija.Surface;
 import org.jfree.graphics2d.Tester;
 import org.jfree.skija.SkijaGraphics2D;
 
 import java.io.IOException;
 
 public class SkijaGraphics2DTestRunner {
 
+    private final static int REPEATS = 100;
+
     /**
      * Run the tests with SkijaGraphics2D.
      *
      * @param fileName  the base filename.
      * @param single  run the current single test?
      */
     public static void testSkijaGraphics2D(String fileName, boolean single) {
-        SkijaGraphics2D g2 = new SkijaGraphics2D((int) Tester.getTestSheetWidth(), (int) Tester.getTestSheetHeight());
-        Tester.drawTestOutput(g2, "SkijaGraphics2D 1.0.4", "https://github.com/jfree/skijagraphics2d", single);
-        io.github.humbleui.skija.Image image = g2.getSurface().makeImageSnapshot();
-        Data pngData = image.encodeToData(EncodedImageFormat.PNG);
-        byte [] pngBytes = pngData.getBytes();
+        final int width = Tester.getTestSheetWidth();
+        final int height = Tester.getTestSheetHeight();
+
+        final SkijaGraphics2D g2 = new SkijaGraphics2D(width, height);
         try {
-            if (single) {
-                fileName += "-single.png";
-            } else {
-                fileName += ".png";
+            for (int i = 0; i < REPEATS; i++) {
+                final long startTime = System.nanoTime();
+
+                Tester.drawTestOutput(g2, "SkijaGraphics2D 1.0.4", "https://github.com/jfree/skijagraphics2d", single);
```

Comment:
Should update the version here - it appears in the sample output.

Reply:
I fixed it in my local copy...

Reply:
You mentioned that you'll do a follow up PR, so just include it in that.  I'll merge this one.